packages/matrix-client-server/src/rooms/room_information/get_joined_rooms.ts

```diff
@@ -0,0 +1,25 @@
+import type MatrixClientServer from '../../index'
+import { send, type expressAppHandler } from '@twake/utils'
+
+export const getJoinedRooms = (
+  clientServer: MatrixClientServer
+): expressAppHandler => {
+  return (req, res) => {
+    clientServer.authenticate(req, res, (data, id) => {
+      const userId = data.sub
```

Comment:
Same, maybe "matrixAddress" could help to understand the value of this

Reply:
You mean "matrixAddress" instead of what exactly ?
here data.sub is used as a userId so it seems adapted

Reply:
`data` became `token`
`const userId = data.sub` could have been written `const matrixAddress = token.sub`